main/signers/index.ts

```diff
@@ -209,24 +213,30 @@ class Signers extends EventEmitter {
       return cb(new Error('Keystores can only be used with ring signers'), undefined)
     }
 
-    ;(signer as RingSigner).addKeystore(keystore, keystorePassword, password, cb)
+    const ringSigner = signer as RingSigner
+    ringSigner.addKeystore(keystore, keystorePassword, password, cb)
   }
 
-  lock(id: string, cb: Callback<Signer>) {
+  lock(id: string, cb: ErrorOnlyCallback) {
     const signer = this.get(id)
 
     // @ts-ignore
     if (signer && signer.lock) {
-      ;(signer as HotSigner).lock(cb)
+      const hotSigner = signer as HotSigner
+      hotSigner.lock(cb)
     }
   }
 
-  unlock(id: string, password: string, cb: Callback<Signer>) {
+  unlock(id: string, password: string, cb: ErrorOnlyCallback) {
     const signer = this.signers[id]
 
     // @ts-ignore
```

Comment:
I think having a comment or TODO associated with each @ts-ignore is a good practise - would be nice to know at a glance if the subsequent code could be fixed with a refactor, if we're waiting on specific types for a dep, etc.

Reply:
agreed, we should always justify them. in this case it was already here but we can add a comment as to why it was put here